src/test/kotlin/org/rust/ide/refactoring/move/RsMoveMultipleFilesTest.kt

```diff
@@ -0,0 +1,104 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.refactoring.move
+
+class RsMoveMultipleFilesTest : RsMoveFileTestBase() {
```

Comment:
Please add a test that we do not handle moving non-Rust files along with Rust ones, like moving `foo.rs` and `Cargo.toml` together

Reply:
Thanks, added (for `.txt` files)